src/scripts/notesGenerator.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-param-reassign, array-callback-return */
+
+function getCommentCount(issueComments, currentTime) {
+  // Filter All comments from the past 24 hours
+  const targetTime = new Date(currentTime);
+  targetTime.setHours(targetTime.getHours() - 24);
+  return issueComments.nodes.filter((comment) => {
+    const commentTime = new Date(comment.updatedAt);
+    return commentTime > targetTime;
+  }).length;
+}
+
```

Comment:
Could you add `getTotalCommits()` count after the Fellowship started? Coz we are adding a leader-board, hopefully

Reply:
Will add separately, needs an entirely different query